packages/mui-base/src/Popper/Popper.test.tsx

```diff
@@ -37,8 +38,7 @@ describe('<Popper />', () => {
       <Popper
         anchorEl={() => document.createElement('div')}
         open
-        component={CustomComponent}
-        ownerState={{ id: 'id' }}
+        slots={{ root: CustomComponent }}
```

Comment:
I am curious, why was the `ownerState` removed? Should it be moved to the `slotProps`? I am not saying to add it back, since it's clearly not needed, but I was wondering if there was another reason

Reply:
Because external `ownerState` isn't supported in Base `Popper` any more. Yes, I think when I removed it, I should've done either `slotProps={{id}}` or just `id={id}`